### DISCUSSION TITLE
How to set npmrc variables via ./configure?

### DISCUSSION THREAD

**Discussion by dkbarn** [nodejs/help#4885] [Q&A]
The documentation for npmrc states that there is a "built-in config file" at ```path/to/npm/itself/npmrc``` and that the variables in this file can be set at build time, via ```./configure```, according to https://docs.npmjs.com/cli/v7/configuring-npm/npmrc#built-in-config-file

However, there are no examples of how to actually do this. And the output from ```./configure --help``` does not show any obvious candidates for setting npmrc variables.

Is the documentation wrong? Or are the ./configure flags hidden?

**Comment by richardlau** [comment]
This is one for @nodejs/npm -- it's referring to https://github.com/npm/cli/blob/latest/configure (note this is a different file from Node.js' `configure` script). This file is present in the npm 6 drops into Node.js, i.e.
- https://github.com/nodejs/node/tree/v12.x-staging/deps/npm
- https://github.com/nodejs/node/tree/v14.x-staging/deps/npm

but missing from the npm 7/8 drops, i.e.
- https://github.com/nodejs/node/tree/v16.x-staging/deps/npm
- https://github.com/nodejs/node/tree/v17.x-staging/deps/npm
- https://github.com/nodejs/node/tree/master/deps/npm
